stdlib/Profile/src/Profile.jl

```diff
@@ -1277,28 +1281,28 @@ backwards-compatibility) and your process is killed, note that this will always
 parts in the same directory as your provided filepath, so you can still reconstruct the
 snapshot after the fact, via `assemble_snapshot()`.
 """
-function take_heap_snapshot(filepath::AbstractString, all_one::Bool=false; streaming::Bool=false)
+function take_heap_snapshot(filepath::AbstractString, all_one::Bool=false, redact_data::Bool=false; streaming::Bool=false)
```

Comment:
From the PR discussion the default should be `true`
```suggestion
function take_heap_snapshot(filepath::AbstractString, all_one::Bool=false, redact_data::Bool=true; streaming::Bool=false)
```

Reply:
Changed to default `true`.